src/common/components/ImageUploader/ImageUploader.js

```diff
@@ -1,36 +1,60 @@
+/* globals cloudmersiveImageApiClient */
+
 import React, { Component } from 'react'
 
 import styles from './ImageUploader.module.scss'
 
+const { REACT_APP_API_KEY } = process.env
+const apiClient = cloudmersiveImageApiClient.ApiClient.instance
+const Apikey = apiClient.authentications['Apikey']
+
 class ImageUploader extends Component {
   constructor (props) {
-    super()
+    super(props)
     this.state = {
-      file: '',
-      imagePreviewURL: ''
+      imagePreviewURL: '',
+      rawFile: ''
```

Comment:
So this is an older pattern for writing React. It is the one in the docs, and you should be familiar with it, but most react projects will look like this:

```
class ImageUploader extends Component {
   // no need to call constructor
   state = { ... }
   
   // no need to bind methods in this component if you use arrow functions
   // because of the way `this` is scoped in arrow functions
   postImageToApi = () => { ... }

   render() {
      return ( )
   }
}
```